src/main/java/capstone/mfslbackend/model/Game.java

```diff
@@ -0,0 +1,52 @@
+package capstone.mfslbackend.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.*;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Objects;
+
+@Getter
+@Setter
+//@RequiredArgsConstructor
+@Entity
+public class Game {
+    @Id
+    Long id;
+    String date;
+    String round;
+
+    public Game() {
+        // Default constructor
+    }
+
+    public void setId(Long id) {
+        this.id = id;
```

Comment:
@Getter and @Setter at the top of the class automatically creates getters and setters.